tests/acceptance/features/cliDataExporter/export.feature

```diff
@@ -74,3 +74,12 @@ Feature: An administrator wants to export the files of his user using
     When user "unknown" is exported to path "/tmp/fooSomething" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "Could not extract user metadata for user"
+
```

Comment:
Please rephrase this 
```
Feature: An administrator wants to export the files of his user using the commandline
```
int0
```
Feature: an administrator wants to export the files of the user using the commandline`
```

Reply:
Please let's not do it in this PR. Refactoring can be done in next

Reply:
please make an issue if we want to do refactoring. So we don't forget it.

Reply:
here's the issue https://github.com/owncloud/data_exporter/issues/226 @nabim777 can work on the refactoring after the merge of this PR